config/default/core.entity_view_display.node.hs_news.hs_vertical_card.yml

```diff
@@ -44,7 +44,7 @@ third_party_settings:
         formatter: default
         settings:
           link: true
-          wrapper: span
+          wrapper: h2
```

Comment:
@pookmish, I'm a little concerned about this one. You and I had talked about how "entity view display and entity form display configs for nodes are ignored," but I dug a little deeper and found this in the config settings:

https://github.com/SU-HSDO/suhumsci/blob/f2721bb8f82d0bbcc981c17ea2996a1479060db8/config/default/config_ignore.settings.yml#L4-L5

It looks to me like only the config for the _default_ view modes are ignored. This isn't a default view mode (it's a hs_vertical_card one). When I look on staging, **it looks like this updated the view mode on existing sites**, which wasn't what we were anticipating. Here's an example:

![image](https://user-images.githubusercontent.com/1256329/73405827-f2a1cc80-42c2-11ea-8872-9d954c8fe8d6.png)

I'm happy to change this back, but how would you feel about updating config-ignore to ignore these other view modes as well like so:

```diff
-  - 'core.entity_view_display.node.hs_*.default'
+  - 'core.entity_view_display.node.hs_*'
```

That would both solve this problem and give us more flexibility in customizing how UI patterns like vertical card work in our theme. Thoughts?

Reply:
this would be fine with me. I didn't notice the change to the display was for a different view mode. I don't believe there would be any harm in ignoring all `hs_*` view displays. Feel free to submit a PR and we can verify the functionality there.
Good thing to notice

Reply:
I have a PR up here: https://github.com/SU-HSDO/suhumsci/pull/461

I'm having some trouble testing it locally because I'm not familiar with the expected behavior for config ignore.

Does `drush cim` recognize the config changes but not apply them? Or is it supposed to not recognize changes made to those files?

Also, I'm trying to figure out how the `local/config_ignore.settings.yml` settings works alongside the settings in `config/default/config_ignore.settings.yml`. Do they both apply? Or just the local ones?

Reply:
during a local `drush cim` the site will only ignore the `local`. All other environments ignore whats in `default`.
The PR looks fine to me. I don't foresee any issues with it.